white_brush/calc_colors.py

```diff
@@ -0,0 +1,30 @@
+from sklearn import cluster
+import numpy as np
+
+
+def choose_representative_colors(colors: np.ndarray):
+    """
+    Read an image from disk and return it as matrix of size (width, height, 3) in the RGB Format.
+    First dimension is the
```

Comment:
@p-hofer 
First dimension is the what?? 😄 
Also the image is not really read from disk, that happened already and the colors are passed in as array

Reply:
hoops, copy paste error ^^